install/kube/cleanup.sh

```diff
@@ -7,6 +7,7 @@ kubectl delete ns $NS
 kubectl delete crd alerts.synopsys.com
 kubectl delete crd blackducks.synopsys.com
 kubectl delete crd opssights.synopsys.com
+kubectl delete crd samples.synopsys.com
```

Comment:
remove the sample crd from cleanup script

Reply:
I removed it